src/figuresClasses.ts

```diff
@@ -1,19 +1,99 @@
 export interface Figure {
-
+  shape: string;
```

Comment:
specify the type more precisely
```
type Shape = 'triangle' | 'circle' | 'rectangle';
```
```suggestion
  shape: Shape;
```

Reply:
Thanks for review, please check the new changes :)